pallets/external-validators-rewards/src/lib.rs

```diff
@@ -115,6 +228,53 @@ pub mod pallet {
             RewardPointsForEra::<T>::remove(era_index_to_delete);
         }
     }
+
+    impl<T: Config> tp_traits::OnEraEnd for Pallet<T> {
+        fn on_era_end(era_index: EraIndex) {
+            if let Some(utils) = Self::generate_era_rewards_utils(era_index, None) {
+                let command = Command::ReportRewards {
+                    timestamp: T::TimestampProvider::get(),
+                    era_index,
+                    total_points: utils.total_points,
+                    tokens_inflated: T::EraInflationProvider::get(),
+                    rewards_merkle_root: utils.rewards_merkle_root,
+                };
+
+                let channel_id: ChannelId = snowbridge_core::PRIMARY_GOVERNANCE_CHANNEL;
+
+                let outbound_message = Message {
+                    id: None,
+                    channel_id,
+                    command,
+                };
+
+                // Validate and deliver the message
+                match T::ValidateMessage::validate(&outbound_message) {
+                    Ok((ticket, _fee)) => {
+                        if let Err(err) = T::OutboundQueue::deliver(ticket) {
+                            log::error!(target: "xcm::ethereum_blob_exporter", "OutboundQueue delivery of message failed. {err:?}");
```

Comment:
this is not ethereum blob exporter, we should change the target

Reply:
_fixed_ in 96da5636be9480807c41c365fb26f9b6c12c2e7a.